packages/keystrokes/src/index.ts

```diff
@@ -70,14 +70,21 @@ export const createTestKeystrokes = <
   OriginalEvent = KeyboardEvent,
   KeyEventProps = MaybeBrowserKeyEventProps<OriginalEvent>,
   KeyComboEventProps = MaybeBrowserKeyComboEventProps<OriginalEvent>
->() => {
+>(
+  options: KeystrokesOptions<
+    OriginalEvent,
+    KeyEventProps,
+    KeyComboEventProps
+  > = {}
+) => {
   let activate: () => void
   let deactivate: () => void
   let press: (event: KeyEvent<OriginalEvent, KeyEventProps>) => void
   let release: (event: KeyEvent<OriginalEvent, KeyEventProps>) => void
 
   const testKeystrokes = Object.assign(
     new Keystrokes<OriginalEvent, KeyEventProps, KeyComboEventProps>({
+      ...options,
```

Comment:
Test harness update was needed in order to pass self releasing keys into test keystrokes instance.

Reply:
A good improvement for sure